### PR TITLE
MNT: Added find_event import to simple_broker.py

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by danielballan** [comment]
Note: we are removing the commandline tests of the examples. They are already tested programmatically, and it's not worth adding complexity to the commandline usage to aim them at a test database.

File: dataportal/tests/test_examples.py [hunk 1]
```diff
@@ -3,9 +3,23 @@
 from ..examples.sample_data import (temperature_ramp, multisource_event,
                                     image_and_scalar)
 from metadatastore.api import Document
+from filestore.utils.testing import fs_setup, fs_teardown
+from metadatastore.utils.testing import mds_setup, mds_teardown
+import uuid
+db_name = str(uuid.uuid4())
```
Comment (tacaswell) [inline dataportal/tests/test_examples.py]: do we still need this?

File: dataportal/tests/test_muxer.py [hunk 2]
```diff
@@ -8,7 +8,20 @@
 from ..sources import switch
 from ..examples.sample_data import (temperature_ramp, multisource_event,
                                     image_and_scalar)
-
+from filestore.utils.testing import fs_setup, fs_teardown
+from metadatastore.utils.testing import mds_setup, mds_teardown
+from metadatastore.api import insert_run_start, insert_beamline_config
+import time as ttime
+
+# def setup():
```
Comment (tacaswell) [inline dataportal/tests/test_muxer.py]: Why don't the setups get called here?

**Comment by ericdill** [comment]
I have no idea why travis is failing on python 3... @tacaswell?

**Comment by tacaswell** [comment]
Interesting.  Given that a weakref is falling over it working in py2k is probably mostly luck of the gc not running right.

**Comment by ericdill** [comment]
Well it's happened on every build so far

**Comment by ericdill** [comment]
Restarted travis after NSLS-II/metadatastore#104. Hopefully this will pass now

**Pull request merged**